### PR TITLE
[ENG-3921] Registries landing page

### PR DESCRIPTION
**Pull request by futa-ikeda**
-   Ticket: [ENG-3921]
-   Feature flag: n/a

## Purpose
- Update registration providers shown on registries landing page
## Summary of Changes
- Add new provider assets for ASIST, Character Lab, egap, Metascience, RWE, YOUth registries' logo
- Add links
- Remove some extra language used in the registration provider list

## Screenshot(s)
image

## Side Effects
- No longer showing language to "Create your own branded registry..."

## QA Notes
- the logos should now link to the given registration provider's discover page (e.g. egap logo will take you to osf.io/registries/egap/discover )

[ENG-3921]: https://openscience.atlassian.net/browse/ENG-3921?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: lib/registries/addon/components/registries-services-list/styles.scss [hunk 1]
```diff
@@ -31,11 +31,27 @@
     background-repeat: no-repeat;
     background-position: center;
 
+    &.provider-asist {
+        background-image: url('img/provider_logos/ASIST_logo.png');
```
Comment (brianjgeiger) [inline lib/registries/addon/components/registries-services-list/styles.scss:35]: I guess this doesn't hurt anything, but why not just put img tags in the template?

**Review by brianjgeiger — CHANGES_REQUESTED** [review]
Could you shrink the logos down to the size that they'll display at? They're not _too_ large, but still.

File: lib/registries/addon/components/registries-services-list/styles.scss (hunk 1 above)
Comment (futa-ikeda) [reply lib/registries/addon/components/registries-services-list/styles.scss:35]: I just followed the pattern that was already there, but using `` is better here

Comment (brianjgeiger) [reply lib/registries/addon/components/registries-services-list/styles.scss:35]: Ah, gotcha.

**Review by brianjgeiger — APPROVED** [review]

**Review by adlius — APPROVED** [review]

**Pull request merged**